README.md

```diff
@@ -4,18 +4,13 @@ The project is inspired by the figma microverse template 1 (https://figma.com/)
 The goal of this project is to learn how parse a Figma design to create a UI, and use Flexbox to place elements in the page.
 
 
+![screenshot](./image/sc-deskt.PNG)
 
-- Screenshoot I
+![screenshot](./image/sc-desktop.PNG)
 
-![screenshot](./image/mob1.PNG)
+![screenshot](./image/sc-desktop1.PNG)
 
-
-- Screenshoot II
-![screenshot](./image/mob2.PNG)
-
-
-- Screenshoot III
-![screenshot](./image/mob3.PNG)
+![screenshot](./image/sc-desktop2.PNG)
 
```

Comment:
- Kindly add the right[ linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) into your pull request.

- Add a description about the changes you made in the pull request description.

Reply:
Hello 
Thank you for your review and comments. 
Please can you explain this part?

Reply:
You are missing steps from the general requirements:
![Screen Shot 2021-07-23 at 07 26 03](https://user-images.githubusercontent.com/22258274/126781433-42201e6a-e111-4619-a727-8cafc0db8ee4.png).

Reply:
Oh I got it thank you.

Reply:
I update it , thank you .

![Descript](https://user-images.githubusercontent.com/6615394/126801100-2b51f829-3b9f-4654-928b-db3252dbe2db.PNG)